token-metadata/program/src/instruction/metadata.rs

```diff
@@ -759,3 +759,87 @@ impl InstructionBuilder for super::builders::Update {
         }
     }
 }
+
+#[repr(C)]
```

Comment:
Since verification has its own section under processor it should have it's own section in instruction IMO.

Reply:
^ Agreed

Reply:
Done